HotelManagement/src/tests/HotelTests.java

```diff
@@ -2,7 +2,6 @@
 
 import core.Hotel;
 import org.junit.jupiter.api.Test;
-
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class HotelTests {
```

Comment:
You need to test all logic in the Hotel class.

You should name your test classes using the class you are testing and suffixing it with Test - HotelTest.